### DISCUSSION TITLE
请问，你们使用时，解暗水印，组件会是深色，挡住了水印的问题吗

### DISCUSSION THREAD

**Discussion by nerveva2000** [zhensherlock/watermark-js-plus#688] [Q&A] [answered]
<img width="1756" alt="zixu" src="https://github.com/zhensherlock/watermark-js-plus/assets/11386193/3b101476-f3bc-4151-8fe7-b25cf3203746">

如上图所示，左侧的黑色区域，完全看不到暗水印，你们又遇到过这种情况，如何处理可以看到暗水印吗？

**Comment by nerveva2000** [comment]
<img width="1756" alt="zixu" src="https://github.com/zhensherlock/watermark-js-plus/assets/11386193/36624ea7-829e-4ec4-b573-c1f8d81b1922">
图片在这里

**Comment by zhensherlock** [comment, marked as answer]
暂时可以使用以下参数来解决深色被挡住的问题
``` js
BlindWatermark.decode({
  compositeOperation: 'overlay',
  compositeTimes: 4,
  url: uploadFile.url,
  onSuccess: (imageBase64) => {
    decodeBlindImageByLight.value = imageBase64
  }
});
```
![image](https://github.com/zhensherlock/watermark-js-plus/assets/3541162/69fa95d4-0784-4a5f-9a10-1ea642bde8f1)

**Reply by zhensherlock** [reply]
文档已更新，可直接在页面上体验
https://zhensherlock.github.io/watermark-js-plus/guide/blind-watermark.html

**Reply by nerveva2000** [reply]
十分感谢，确实有效果

最稳一个问题，我在做另外一种测试的时候，发现，如果页面上有一块大图片，图片会变的非常黑，图片无法识别到内部暗水印信息，这方面参数如何调整，可以让图片中的水印显示的更清晰

这是通过网站解析得到的结果
<img width="1749" alt="image" src="https://github.com/zhensherlock/watermark-js-plus/assets/11386193/4c2b591e-1142-46d2-b06b-7268967c7d31">


这是我的原图
<img width="1745" alt="截屏2024-07-05 00 03 06" src="https://github.com/zhensherlock/watermark-js-plus/assets/11386193/982b4ebe-6138-4a91-a86c-95db568792ae">


如果有时间，给我一些建议，谢谢

**Reply by zhensherlock** [reply]
暂时还没有找到针对图片上水印的合适参数
